source/fuzz/transformation_replace_constant_with_uniform.cpp

```diff
@@ -90,6 +90,40 @@ TransformationReplaceConstantWithUniform::MakeLoadInstruction(
                                       operands_for_load);
 }
 
+opt::Instruction*
+TransformationReplaceConstantWithUniform::GetInsertBeforeInstruction(
+    opt::IRContext* ir_context) const {
+  auto* result =
+      FindInstructionContainingUse(message_.id_use_descriptor(), ir_context);
+  if (!result) {
+    return nullptr;
+  }
+
+  // OpPhi must be the first instruction in the module. Thus, we can't insert
+  // anything above it.
```

Comment:
I think this should say something like:


The use might be in an OpPhi instruction.


Then, within the if body:


OpPhi instructions must be the first instructions in a block. Thus, we can't insert above the OpPhi instruction. Given the predecessor block that corresponds to the id use, get the last instruction in that block above which we can insert OpAccessChain and OpLoad.

Reply:
Fixed in bcb5394.